packages/container/lib/container.js

```diff
@@ -314,24 +314,15 @@ function areInjectionsNotDynamic(injections) {
   return injections._dynamic !== true;
 }
 
-function buildInjections() /* container, ...injections */{
+function buildInjections(container, injections) {
```

Comment:
Shouldn't it be `...injections` here?

Reply:
the point of PR was to avoid touching arguments list :) and passing array instead, `...injections` will compile to again argument touching code I think :P

Reply:
It will not if there is a parameter first.

Reply:
Example: http://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&code_lz=GYVwdgxgLglg9mABACzgGwIYAoB0eZgBWAptPGAM4CUiA3gFCKIQIXrE5pwDmu-RpWKyr0AvvXqhIQpKkxYMAGkR4cBEmWF1GzVu048-agZspUA3GKA&debug=false&circleciRepo=&evaluate=false&lineWrap=true&presets=es2015%2Creact%2Cstage-2&prettier=false&targets=&version=6.26.0

Reply:
in our case it will :P, because there is for loop, http://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&code_lz=GYVwdgxgLglg9mABACzgGwIYAoB0eZgBWAptPGAM4CUiA3gFCKIQIXrE5pwDmu-RpWKyr0AvvXqhIQpKkxYMAGkR4cBEmWF1GzVu048-agZspUA3GIlTTKdNiUr-GmdW1NgcAE5YAbhi9EGEQAXkQABnMggB5EAEZImABqJKoGJgBIFkp9Ll51QXIKAG0YAF0LDJ1xUSA&debug=false&circleciRepo=&evaluate=false&lineWrap=true&presets=es2015%2Creact%2Cstage-2&targets=&version=6.26.0

Reply:
it is much better to pass just array :P